docs/index.md

```diff
@@ -1,21 +1,24 @@
 # Exploring COVID-19 Vaccine Effectiveness
+This project is an exploratory data analysis (EDA) of COVID-19 vaccine effectiveness. The team decided to examine commonly used COVID-19 information sources (Center For Disease Control and Johns Hopkins University) and visualize the relationship between Covid-19 vaccinations and deaths the United States between 6/30/21 - 11/30/21. The visualization below represents this analysis.
```

Comment:
The date range for the final scatter plots is 6/30/21 - 11/30/21. This was done for visual/style purposes. The county-specific bar chart date range spans an extra month between 5/31/21 - 11/30/21. 

We have a few options. 
1. Remove the analysis date in this beginning overview because the charts are clearly labeled with the dates. 
2. Update the analysis date in this beginning overview from 5/31/21 - 11/30/21. 
3. Add the additional scatter plot to the scatter visuals which impact visualization. 

I recommend we update the date in this beginning overview to 5/31/21 - 11/30/21 since the visuals below are clearly labeled and defined. Thoughts?

Reply:
It would be cleaner if we hyperlink the data sources for (the Center For Disease Control and Johns Hopkins University) in this section, include the last sentence "For more information visit the repository ..." here, and delete the about the data section. Thoughts?

Reply:
> The date range for the final scatter plots is 6/30/21 - 11/30/21. This was done for visual/style purposes. The county-specific bar chart date range spans an extra month between 5/31/21 - 11/30/21.
> 
> We have a few options.
> 
> 1. Remove the analysis date in this beginning overview because the charts are clearly labeled with the dates.
> 2. Update the analysis date in this beginning overview from 5/31/21 - 11/30/21.
> 3. Add the additional scatter plot to the scatter visuals which impact visualization.
> 
> I recommend we update the date in this beginning overview to 5/31/21 - 11/30/21 since the visuals below are clearly labeled and defined. Thoughts?

Yes I agree that we should change this to 5/31/21, esp. because now we will have the actual animated gif with all the data rather than it starting in June.

Reply:
> It would be cleaner if we hyperlink the data sources for (the Center For Disease Control and Johns Hopkins University) in this section, include the last sentence "For more information visit the repository ..." here, and delete the about the data section. Thoughts?

Great idea, I fixed this as well.